code/game/turfs/turf.dm

```diff
@@ -213,6 +213,10 @@
 	if(!GLOB.use_preloader && path == type) // Don't no-op if the map loader requires it to be reconstructed
 		return src
 
+	if(isspaceturf(path) && is_mining_level(z)) //if we are trying to make space on lavaland, we must scream and redirect
+		log_debug("Something just tried to turn [src] into space on lavaland, report it to a coder. [ADMIN_COORDJMP(src)]")
```

Comment:
I'd think coders would appreciate a stacktrace? (maybe in addition to the `log_debug`? (actually do admins on live have `log_debug` enabled even?..))

Reply:
I do, but I know not many people do